packages/pipelines-v5/commands/pipelines/diff.js

```diff
@@ -46,18 +46,19 @@ function * getAppInfo (heroku, appName, appId) {
   // Find the commit hash of the latest release for this app
   let slug
   try {
-    const release = yield heroku.request({
+    const releases = yield heroku.request({
       method: 'GET',
       path: `/apps/${appId}/releases`,
-      headers: { 'Accept': V3_HEADER, 'Range': 'version ..; order=desc,max=1' },
+      headers: { 'Accept': V3_HEADER, 'Range': 'version ..; order=desc' },
       partial: true
     })
-    if (release[0].slug === null) {
+    const release = releases.filter((r) => r.status === 'succeeded')[0]
```

Comment:
`release` can still end up as `null` here, if none of the previous releases had succeeded (or none within the window of the set of releases returned by the upstream service).

Reply:
Yup, but afaik, the cli doesn't give us any way to manage pagination.

Reply:
You could probably use this instead...

```JavaScript
    const release = releases.find((r) => r.status === 'succeeded')
```

But that feels like optimisation rather than a blocker.